shopinvader/models/shopinvader_backend.py

```diff
@@ -82,6 +82,24 @@ class ShopinvaderBackend(models.Model):
     website_public_name = fields.Char(
         help="Public name of your backend/website."
     )
+    clear_cart_options = fields.Selection(
+        selection=[
+            ("delete", "Delete"),
+            ("clear", "Clear"),
+            ("new_cancel", "New (cancel previous)"),
```

Comment:
@acsonefho I'm gonna be bored again but I would prefer 'cancel' in place of 'new_cancel' 
```suggestion
            ("cancel", "Cancel"),
```